tests/cargo-ui/ws-package-exclude-unknown/src/main.rs

```diff
@@ -0,0 +1,9 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+fn main() {}
+
+#[kani::proof]
+fn harness_in_ws_package() {
+    assert!(1 + 1 == 2);
```

Comment:
Perhaps additionally make the assertion incorrect (e.g. `1 + 1 == 3`) so that verification fails if the package is incorrectly excluded.

Reply:
That's a great idea, thanks!